reports/Oklahoma.md

```diff
@@ -0,0 +1,11 @@
+## Oklahoma City
+
+### Police fire projectile directly at the chest of someone standing with hands up  | May 30th
+
+Police fire projectile directly at the chest of someone who is standing with hands up. You can see other projectiles being fired straight at other protestors.
```

Comment:
Video does not show who shot projectile - which is likely a rubber bullet, so I think it's more correct to state "person with hands up struck in chest by projectile; a second projectile is fired shortly after person is struck by the first"

Reply:
Thanks, good suggestion